docs/notebooks/0_introduction.ipynb

```diff
@@ -29,17 +36,11 @@
     "    - [How to upload a data product to the PZ Server](#how-to-upload-a-data-product-to-the-pz-server)\n",
```

Comment:
I don't know if it is a local problem, but the links for the "How to upload a data product to the PZ Server", 
"How to download a data product from the PZ Server", "How to get general info from PZ Server" and "How to display the metadata of a data product" sections are not taking me to the corresponding secions in the notebook.